app/Services/MarketDataProviders/CoinGecko.php

```diff
@@ -107,19 +107,40 @@ public function priceAndPriceChange(string $baseCurrency, Collection $targetCurr
 
     private function isEmptyResponse(?array $data): bool
     {
-        if ($data === null) {
-            $times = Cache::increment('coin_gecko_response_error');
+        return $this->isAcceptableResponse(
+            $data,
+            'coin_gecko_response_error',
```

Comment:
🤔 can we rename this to `coingecko_response_error`? it's not two words and brings it in line with the config. Same for the throttles cache key

Reply:
yeh i shall update it